webapp/src/components/HomePage/Map/ParcelsMap/ParcelsMap.js

```diff
@@ -242,15 +258,17 @@ export default class ParcelsMap extends React.Component {
     }
   }
 
-  // Called by the Parcel Grid on each tile render
-  getTileAttributes = latlng => {
-    const { wallet, parcels, districts } = this.props
+  getTileAttributesFromLatLng = latlng => {
     const { x, y } = this.mapCoordinates.latLngToCartesian(latlng)
+    const { wallet, parcels, districts } = this.props
+    return this.getTileAttributes(x, y, wallet, parcels, districts)
+  }
+  // Called by the Parcel Grid on each tile render
+  getTileAttributes = (x, y, wallet, parcels, districts) => {
```

Comment:
correct me if I'm wrong, but now that we have `getTileAttributesFromLatLng` we can change the firm of `getTileAttributes` to be `getTileAttributesFromLatLng = (x, y) {` to avoid having to pass 5 arguments and removing the repeated `const { wallet, parcels, districts } = this.props` on top of each call